lib/cf_app_discovery/target.rb

```diff
@@ -14,7 +14,11 @@ def initialize(guid:, name:, instances:, state:, route:, space:, org:, detected_
     end
 
     def paas_metric_exporter?
-      detected_start_command == "./bin/paas-metric-exporter"
+      # This flag is used as a indicator that the target running is either the paas-metric-exporter or the
+      # paas-prometheus-exporter. These two apps both require special treatement when generating their json.
+      # This function can likely be renamed to `paas-prometheus-exporter?` after prometheus support is removed from
```

Comment:
```suggestion
      # This function can likely be renamed to `paas_prometheus_exporter?` after prometheus support is removed from
```
can't use kebab-case in ruby identifiers